airbyte-server/src/main/java/io/airbyte/server/converters/JobConverter.java

```diff
@@ -137,7 +137,7 @@ public AttemptInfoRead getAttemptInfoRead(final Attempt attempt) {
 
   public static AttemptRead getAttemptRead(final Attempt attempt) {
     return new AttemptRead()
```

Comment:
I will also change the api in a later PR to a better name. Waiting to sync up with the FE engineers first.